docs/how_to_guides/how_to_create_surrogate_models.rst

```diff
@@ -0,0 +1,179 @@
+How to create a surrogate model
```

Comment:
```suggestion
How to create a surrogate model class
```
This seems more like "how to create a surrogate model class" because demonstrating a surrogate model could be done more succinctly and simply. 

I wonder if you should also add a one to two sentence preamble explaining how we've approached the creation of surrogate models for power systems (maybe doesn't have to be that specific, but some context to what's done in REFLO and why might be useful).

Reply:
Yeah that's true, I wanted the how-to to closely reflect our full implementation. I'll add some context on this next commit